src/Razor/src/Microsoft.CodeAnalysis.Remote.Razor/AutoInsert/OOPAutoInsertService.cs

```diff
@@ -0,0 +1,16 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT license. See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using System.Composition;
+using Microsoft.CodeAnalysis.Razor.AutoInsert;
+
+namespace Microsoft.CodeAnalysis.Remote.Razor.AutoInsert;
+
+[Export(typeof(IAutoInsertService)), Shared]
+[method: ImportingConstructor]
+internal sealed class OOPAutoInsertService(
+    [ImportMany]IEnumerable<IOnAutoInsertProvider> providers
+    ) : AutoInsertService(providers)
```

Comment:
Please move closing parenthesis to the previous line. Although, this probably isn't long enough to warrant splitting across lines anyway. Also, please add a space between `[ImportMany]` and `IEnumerable`.
 
```suggestion
internal sealed class OOPAutoInsertService([ImportMany] IEnumerable<IOnAutoInsertProvider> providers) : AutoInsertService(providers)
```